examples/hf_serialization.py

```diff
@@ -60,23 +110,40 @@ def serialize_model(
             is purely optional, and it allows for multiple models to be
             serialized to the same directory. A good example are Stable
             Diffusion models. Default is "model".
+        force: Force upload serialized tensors to `output_prefix`
+            even if they already exist
     """
 
-    os.makedirs(model_directory, exist_ok=True)
-    dir_prefix = f"{model_directory}/{model_prefix}"
+    if model_prefix is None:
+        model_prefix = "model"
 
+    dir_prefix = f"{model_directory}/{model_prefix}"
+    config_file_exists, weights_file_exists = (
+        check_file_exists(f"{dir_prefix}-config.json"),
+        check_file_exists(f"{dir_prefix}.tensors"),
+    )
```

Comment:
There should be an error case if only one of these exists and `force` is not enabled, since selectively writing one of them will quite possibly break something. This might occur if they manually deleted the `.tensors` file and forgot to delete the `-config.json` file.

Reply:
Good point. Will handle this.

Reply:
This should hopefully be resolved with the most recent commit.